openstack/clustering/v1/webhooks/results.go

```diff
@@ -0,0 +1,31 @@
+package webhooks
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// GetResult is the response of a Get operations. Call its Extract method to
+// interpret it as a Version.
+type GetResult struct {
+	commonResult
+}
```

Comment:
I think we can safely remove `GetResult` here since this PR is only defining a `Trigger` action.

Reply:
Removed